### PR TITLE
Added remove by age endpoint test in support-logging

### PR DESCRIPTION
**Pull request by jduranf**
Implements #142. Don't merge until https://github.com/edgexfoundry/edgex-go/pull/1062 have been merged.

Signed-off-by: Joan Duran <jduran@circutor.com>

### CODE REVIEW

**Comment by cloudxxx8** [comment]
@cherrycl please help review this change

File: bin/postman-test/collections/support-logging.postman_collection.json [hunk 1]
```diff
@@ -1241,6 +1241,101 @@
...
+							"path": [
+								"api",
+								"v1",
+								"logs",
+								"removeold",
+								"age",
+								"{{age}}"
+							]
+						},
+						"description": "Delete all log entries older than the specified time. HTTP status code 200 count of the number of LogEntry being deleted. HTTP status code 503 for unknown or unanticipated issues."
```
Comment (tsconn23) [inline bin/postman-test/collections/support-logging.postman_collection.json]: Error HTTP status code should be `500` instead of `503`

**Comment by tsconn23** [comment]
I'm planning to merge https://github.com/edgexfoundry/edgex-go/pull/1062 shortly. I ran the test via Postman.

File: bin/postman-test/collections/support-logging.postman_collection.json [hunk 2]
```diff
@@ -1241,6 +1241,101 @@
...
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "3e7841f2-239d-11e9-ab14-d663bd873d93",
+								"exec": [
+									" /**",
+									" * Test Case:  /api/v1/logs/removeold/age/{age} - DELETE",
+									" * Version: Alpha",
+									" * @Author: Tom Fleming",
```
Comment (cherrycl) [inline bin/postman-test/collections/support-logging.postman_collection.json]: Please replace the author name.

File: bin/postman-test/collections/support-logging.postman_collection.json (hunk 1 above)
Comment (jduranf) [reply bin/postman-test/collections/support-logging.postman_collection.json]: Changed all `503` to `500`. Anyway, all descriptions are wrong. The endpoint can return a `400`, which is not described. Maybe the best option is to remove from the description the expected return values.
Also the [raml](https://github.com/edgexfoundry/edgex-go/blob/master/api/raml/support-logging.raml) file should be modified. This file doesn't define the `400`, and defines `413`, which is never returned.
But let's do those change in others PR.

File: bin/postman-test/collections/support-logging.postman_collection.json (hunk 2 above)
Comment (jduranf) [reply bin/postman-test/collections/support-logging.postman_collection.json]: Name replaced

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**